### PR TITLE
Added a check for max_age being greater than 0

### PR DESCRIPTION
**Pull request by caitlin-tibbetts**
Issue #147 

Added a check for max_age being greater than 0 and added an appropriate error message.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by 18AdrianoH** [comment]
Please write tests.

**Comment by caitlin-tibbetts** [comment]
> 
> 
> Please write tests.

Done.

**Comment by karlw00t** [comment]
Is there any documentation that needs to be updated for this change?

**Comment by caitlin-tibbetts** [comment]
> Is there any documentation that needs to be updated for this change?

No, the [documentation](https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/thresholds.html) already says that the maximum age must be "a value greater than 0," so there is no update needed.

File: test/unit/test_material_managers_caching.py [hunk 1]
```diff
@@ -117,6 +117,7 @@ def test_mkp_to_default_cmm(mocker):
             dict(max_bytes_encrypted=MAX_BYTES_PER_KEY + 1),
             r"max_bytes_encrypted cannot exceed {}".format(MAX_BYTES_PER_KEY),
         ),
+        (dict(max_age=0.0), r"max_age cannot be less than or equal to 0"),
```
Comment (mattsb42-aws) [inline test/unit/test_material_managers_caching.py]: For completeness, let's also add a negative value test case.

**Comment by mattsb42-aws** [comment]
AppVeyor failures are due to known issue #159.

**Pull request merged**